content/reference/sql/version-control/dolt-sql-functions.md

```diff
@@ -554,6 +554,7 @@ The `DOLT_LOG()` table function takes up to two optional revision arguments:
   Note: the CLI `dolt log` command defaults to "short", while this table function defaults
   to "no".
 - `--not`: Excludes commits reachable by revision.
+- `--tables`: Limits the log to commits that affect the specified tables. Any number of tables can be specified.
```

Comment:
You should specify that table names are comma separated. The way it sounds now is that you can give the --table argument multiple times.